pennylane/__init__.py

```diff
@@ -16,47 +16,46 @@
 PennyLane can be directly imported.
 """
 from importlib import reload
-import pkg_resources
 
 import numpy as _np
-
-from semantic_version import Version, Spec
-
-# QueuingContext needs to be imported before all other pennylane imports
-from ._queuing import QueuingContext  # pylint: disable=wrong-import-order
-import pennylane.operation
+import pkg_resources
```

Comment:
Don't forget to fix this one.

Reply:
@mariaschuld what do you mean by this comment?

Reply:
Sorry, I meant the codefactor comment... :)

Reply:
Ah, good catch - this is because code factor is getting confused between `pennylane.numpy` and vanilla numpy. I ran pylint locally and this doesn't occur, so I _think_ it is a codefactor issue?